javascript/javascript1/week2/weather_wear.js

```diff
@@ -0,0 +1,23 @@
+function choosenCloses(temperature) {
+    let clothesToWear;
+    if (temperature < -10) {
+        clothesToWear = "Your clothes must be very, very warm";
+    }
+    if (-10 <= temperature >= 0) {
+        clothesToWear = "Your clothes should be very warm";
+    }
+    if (1 <= temperature >= 7) {
+        clothesToWear = "Your clothes should be warm coat, hat, hoody";
+    }
+    if (8 <= temperature >= 15) {
+        clothesToWear = "Your clothes should be windbreaker, sweater";
+    }
+    if (temperature >= 16) {
+        clothesToWear = "Your clothes should be shorts and t-shirts";
+    }
```

Comment:
You can't really chain the comparison operators like this. 
You have to combine them with `&&` or `||`.
`if(-10 <= temperature && temperature <= 0) {`

Reply:
You've flipped the second comparison in each statement:
```javascript
if (-10 <= temperature >= 0) { // this will return true for any temperature above 0
```

Reply:
When you do if statements like these that cover each other's cases, you can simplify it like this:
```javascript
if (temperature < -10) {
  clothesToWear = "Your clothes must be very, very warm";
} else if (temperature <= 0) {
  clothesToWear = "Your clothes should be very warm";
} else if (temperature <= 7) {
  clothesToWear = "Your clothes should be warm coat, hat, hoody";
} else if (temperature <= 15) {
  clothesToWear = "Your clothes should be windbreaker, sweater";
} else { // It's hotter than 15 degrees
  clothesToWear = "Your clothes should be shorts and t-shirts";
}
```

Reply:
Sure, this is better.